pkg/lvgl/include/lvgl_riot.h

```diff
@@ -33,9 +33,9 @@ extern "C" {
 void lvgl_init(screen_dev_t *screen_dev);
 
 /**
- * Start the lvgl task handler background thread
+ * Run the lvgl task handler
 */
-void lvgl_start(void);
+void lvgl_run(void);
```

Comment:
Maybe mention the usage of this function on the `pkg/lvgl/doc.txt` file? So possible users know that they have to manually run the task handler.

Reply:
Done